app/styles/components/_c-infinite-list.scss

```diff
@@ -0,0 +1,7 @@
+.au-c-infinite-list {
+  > li + li {
+    margin-top: $au-unit;
```

Comment:
Possibly replaceable using au- classes

Reply:
I prefer defining it here as the AU list doesn't cover the design spec. Will see if I can extend the list component in the future.